platform/ios/src/MGLMapView.h

```diff
@@ -740,6 +740,10 @@ MGL_EXPORT
  */
 @property(nonatomic, getter=isPitchEnabled) BOOL pitchEnabled;
 
+/**
+ A Boolean value that determines whether the user can change the center coordinate of the map while rotating or zooming. When this property is set to YES, the default, the location of the gesture in the map view determines the map's center coordinate.
+ */
+@property(nonatomic) BOOL allowScrollGesturesDuringRotateOrZoom;
```

Comment:
I think the naming of this property does not reflect the real purpose. I was thinking if we could change this to `isCenterCoordinateGestureAnchor` with a default value = `NO`

Reply:
This name is same as Google Maps, I think we don't need to change it

Reply:
Hi. Thank you for your reply. Although Google is using something similar is also misleading. Perhaps:
```
anchorRoteOrZoonGesturesToCenterCoordinate
```
Communicates better that when turned on it will use center coordinate as anchor.

Reply:
Did you mean `anchorRotateOrZoomGesturesToCenterCoordinate` ?
It sounds better!

Reply:
Hi. Yes, I mean that. Fixing my typos.

Reply:
😬  Do you have comment about it's description?

Reply:
> A Boolean value that determines whether gestures are anchored to the center coordinate of the map while rotating or zooming. 
Default value is set to `NO`.

Reply:
fixed